src/main/java/org/jitsi/jigasi/transcription/Transcript.java

```diff
@@ -0,0 +1,427 @@
+/*
+ * Jigasi, the JItsi GAteway to SIP.
+ *
+ * Copyright @ 2017 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.jigasi.transcription;
+
+import java.time.*;
+import java.time.format.*;
+import java.util.*;
+
+/**
+ * A transcript of a conference. An instance of this class will hold the
+ * complete transcript once a conference is over
+ *
+ * @author Nik Vaessen
+ */
+public class Transcript
+    implements TranscriptionListener
+{
+    /**
+     * Formats an Instant to <hour:minute:second> in the timezone the machine
+     * is running on
+     */
+    private static final DateTimeFormatter timeFormatter
+        = DateTimeFormatter
+        .ofLocalizedTime(FormatStyle.MEDIUM)
+        .withZone(ZoneId.systemDefault());
```

Comment:
I think we should either use UTC, or explicitly include the timezone somewhere. What do you think?